source/common/grpc/google_grpc_context.cc

```diff
@@ -7,15 +7,19 @@
 #include "common/common/macros.h"
 #include "common/common/thread.h"
 
+#ifdef ENVOY_GOOGLE_GRPC
```

Comment:
Sorry why is this needed again? I thought we are not including this class unless Google gRPC is enabled? @jmarantz @lizan ?

Reply:
I did not understand why this was needed either. When I approved this PR this change had not been made yet.

Reply:
@qiwzhang can you do a follow up to revert this part? Thank you.

Reply:
Sorry,  #8407 will revert this change